nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 12 Line 3_
```markdown
## Square bracket search []

Get a single record by entering a primary key into a table object within square brackets. 
```
Can also be multiple values for composite keys.